modules/zivid/capture_assistant.py

```diff
@@ -0,0 +1,96 @@
+"""Contains capture assistant functions and classes."""
+from enum import Enum
+
+import _zivid
+import zivid._settings_converter as _settings_converter
+
+
+_AmbientLightFrequencyEntries = {  # pylint: disable=invalid-name
+    entry: entry
+    for entry in _zivid.capture_assistant.AmbientLightFrequency.__entries  # pylint: disable=protected-access
+}
+_AmbientLightFrequencyToInternalMap = {  # pylint: disable=invalid-name
+    entry: getattr(_zivid.capture_assistant.AmbientLightFrequency, entry)
+    for entry in _AmbientLightFrequencyEntries
+}
+AmbientLightFrequency = Enum(  # pylint: disable=invalid-name
```

Comment:
Simple hard-coded and checked implementation may be:
```
class AmbientLightFrequency(Enum):
    hz50 = "hz50"
    hz60 = "hz60"
    none = "none"

    def __str__(self):
        return str(self.name)


assert AmbientLightFrequency.__members__.keys() == _zivid.captureassistant.AmbientLightFrequency.__members__.keys(), f"{AmbientLightFrequency} not in sync with C++ implementation"

```

Reply:
The above may be improved (e.g. to avoid hz50 = "hz60"):

```
class StrEnumMeta(EnumMeta):
    def __new__(metacls, cls, bases, oldclassdict):
        newclassdict = _EnumDict()
        for k, v in oldclassdict.items():
            if v == ():
                v = k
            newclassdict[k] = v
        return super().__new__(metacls, cls, bases, newclassdict)

class AutoStrEnum(str, Enum, metaclass=StrEnumMeta):
    "base class for name=value str enums"

class AmbientLightFrequency(AutoStrEnum):
    hz50 = ()
    hz60 = ()
    none = ()

    def __str__(self):
        return str(self.name)

assert AmbientLightFrequency.__members__.keys() == _zivid.captureassistant.AmbientLightFrequency.__members__.keys(), f"{AmbientLightFrequency} not in sync with C++ implementation"
```

Reply:
Another implementation:

- automatically in sync with values exposed by pybind11

- but, could not make __str__ work without dynamically adding the method inlater step

- unsure if auto-completion works everywhere (does in IPython)

´´´
class _AmbientLightFrequencyEnumMeta(EnumMeta):

    def __new__(metacls, cls, bases, oldclassdict):
        newclassdict = _EnumDict()
        for member in _zivid.captureassistant.AmbientLightFrequency.__members__:
            newclassdict[member] = str(member)
        return super().__new__(metacls, cls, bases, newclassdict)


class AmbientLightFrequency(str, Enum, metaclass=_AmbientLightFrequencyEnumMeta):
    "base class for name=value str enums"


setattr(AmbientLightFrequency, "__str__", lambda self: str(self.name))
´´´